playbooks/gateway-database-export.yml

```diff
@@ -1,7 +1,20 @@
 ---
 # Runs mysql to get gateway and OTK database dump against all the hosts and uploads the results into downloads folder (set by default)
+- name: Detect source gateway mysql nodes
+  hosts: gateway_mysql
+  gather_facts: no
+  tasks:
+    - name: collect src hosts
+      add_host:
+        hostname: "{{ hostvars[item].src }}"
+        ansible_connection: "ssh"
+        groups:
+          - gateway_mysql
```

Comment:
Should "gateway_mysql" be here? It is already defined in hosts file and here it adds more to it?

Reply:
yes basically we are dynamically creating hosts since we only want src machines. so we have to add it to the group dynamically.